### DISCUSSION TITLE
New Feature: Confidence Rating

### DISCUSSION THREAD

**Discussion by TheTechromancer** [blacklanternsecurity/bbot#975] [Ideas]
The idea came up during BBOT Con of assigning a "confidence" score to events. This would speed up the process analysts go through to confirm assets.

@SpamFaux @aconite33 @batgoose @mgaddy

Factors might include:
1. IP address
     - Whether the asset is in-scope because of its hostname or IP address (hostnames that resolve to in-scope IPs would have lower confidence that in-scope hostnames)
     - Cloud provider (registrar IPs suggest parked pages == lower confidence)
2. Domain
     - Registrar
3. Source module
    - Active modules == higher confidence

**Comment by liquidsec** [comment]
Implemented in 3.0
